README.md

```diff
@@ -562,11 +604,11 @@ packages:
   - package: fivetran/linkedin_source
     version: [">=0.7.0", "<0.8.0"]
 
-  - package: fivetran/twitter_ads
-    version: [">=0.6.0", "<0.7.0"]
+  - package: fivetran/reddit_ads
```

Comment:
Won't we need to update all of these?

Reply:
Bumping this comment as it looks like not all dependencies have been updated yet.

Reply:
Sorry about that. Updated!